tribits/CHANGELOG.md

```diff
@@ -4,10 +4,23 @@ ChangeLog for TriBITS
 
 ## 2022-03-02:
 
-* **Added:** Added project-level cache varaible `<Project>_IMPORTED_NO_SYSTEM`
-  to set the `IMPORTED_NO_SYSTEM` property on the exported IMPORTED library
-  targets in the installed `<Package>Config.cmake` files (see updated TriBITS
-  documentation for `<Project>_IMPORTED_NO_SYSTEM`).
+* **Added:** The project-level cache variable `<Project>_IMPORTED_NO_SYSTEM`
+  was added to set the `IMPORTED_NO_SYSTEM` property (CMake versions 3.23+
+  only) on the exported IMPORTED library targets in the installed
+  `<Package>Config.cmake` files (see updated TriBITS users guide and build
+  reference documentation for `<Project>_IMPORTED_NO_SYSTEM`).  Setting this
+  to `ON` results in the include directories for this project's IMPORTED
+  library targets to be listed on the compile lines in downstream CMake
+  projects using `-I` instead of the default `-isystem` for IMPORTED library
+  targets.  Therefore, setting this option to `ON` returns backward
+  compatibility for the move to modern CMake targets which involved setting
+  the include directories on the IMPORTED library targets using
+  `target_include_directories()` described below (which changed the include
+  directories from being listed as `-I` to `-isystem` by default).<br>
```

Comment:
The behavior of the project-level cache variable `<Project>_IMPORTED_NO_SYSTEM` is not defined here in case of a user try to set it with a CMake version < 3.23

Reply:
No, but the impact of trying to set `<Project>_IMPORTED_NO_SYSTEM=ON` for CMake versions less than 3.23 is documented in the build reference entry which is referred to by  the above statement "see updated TriBITS users guide and build reference documentation for `<Project>_IMPORTED_NO_SYSTEM`".  I am just trying to reduce the documentation duplication some (but there is still a lot of duplication as it is).  If any user makes the mistake to set this to `ON` with CMake versions less than 3.23, they will get a fatal error message about there not being support.